src/index.html

```diff
@@ -67,9 +67,9 @@
   <body>
     <%~ includeFile("partials/google-analytics-noscript.html") %>
     <a class="visually-hidden visually-hidden-focusable" href="#content">
-      Skip to main content
+      <%= it.t.skip_to_main %>
     </a>
-    <%~ includeFile("partials/nav.html", { page: "index" }) %>
+    <%~ includeFile("partials/nav.html", { page: "index", t: it.t.nav, prefix: it.prefix }) %>
```

Comment:
I like how `t` gets recursively passed around here. It looks like a nice abstraction.

Reply:
In theory such 'folding' can also be applied to `renderTemplate`, but we still need some shared strings between pages.